### PR TITLE
Add Jquery and Jquery UI webjars

### PR DESCRIPTION
**Pull request by litvinovg**
# What does this pull request do?
Added webjars with latest jquery and jquery ui versions
Added datatables dependency used by Map of Science 
https://mvnrepository.com/artifact/org.mvnpm/datatables.net/2.0.5
Added jquery migrate dependency
https://mvnrepository.com/artifact/org.webjars.npm/jquery-migrate/3.4.1
All dependencies are licensed under the MIT license.

# How should this be tested?
Build Vitro and VIVO

# Interested parties
@gneissone 

# Reviewers' expertise

Candidates for reviewing this PR should have some of the following expertises:
1. Maven

# Reviewers' report template
## General comment
A reviewer should provide here comments and suggestions for requested changes if any.
## Testing
A reviewer should briefly describe here how it was tested
## Code reviewing
A reviewer should briefly describe here which part was code reviewed

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -38,6 +121,26 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.webjars.npm</groupId>
+            <artifactId>jquery-migrate</artifactId>
+            <version>3.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mvnpm</groupId>
```
Comment (chenejac) [inline webapp/pom.xml:130]: @litvinovg did you investigate difference between this artifact in org.mvnpm and org.webjars (https://mvnrepository.com/artifact/org.webjars.npm/datatables.net/2.0.5)?

**Review by chenejac — CHANGES_REQUESTED** [review]
## General comment
@litvinovg thanks a lot for this PR. 

## Testing
I have run building of VIVO and webjars directory inside webapps/target directory has been created. 

## Code reviewing
I have posted one comment regarding group id for datatables.net artifact. I am also wondering whether we need all those dependencies extracted into webjars directory, or we can use existing js, and css directories?

File: webapp/pom.xml (hunk 1 above)
Comment (litvinovg) [reply webapp/pom.xml:130]: As you can see on this page version 2.0.5 is still not available for org.webjars
https://mvnrepository.com/artifact/org.webjars/datatables

Comment (chenejac) [reply webapp/pom.xml:130]: let's keep group id which is set at the moment

**Review by chenejac — APPROVED** [review]

**Comment by hudajkhan** [comment]
.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by wwelling — APPROVED** [review]

**Pull request merged**